lang/cs/Org.Apache.REEF.Client/YARN/YarnREEFParamSerializer.cs

```diff
@@ -161,7 +161,7 @@ internal byte[] SerializeJobArgsToBytes(JobParameters jobParameters, string driv
 
             var avroYarnClusterJobSubmissionParameters = new AvroYarnClusterJobSubmissionParameters
             {
-                //// securityTokenKind and securityTokenService will be deprecated 
+                //// TODO: [JIRA REEF-1887] Deprecated. Will remove.
```

Comment:
use simple `//` comment; here and everywhere: specify REEF **version** in which it should be removed